docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4550,7 +4566,11 @@ The egress packets may be distinguished for further processing in the egress
 using the `instance` metadata. Note that a packet may not be both unicast and
 multicast; if the multicast group is set, it will override the unicast egress
 port. If the P4 `multicast_group` metadata is set to a value that is not
-programmed in the PRE, then the packet is dropped.
+programmed in the PRE, then the packet is dropped. For each replicas(i) in
+the `multicast_group_entry`, there must be an associated backups(i) or
+`multicast_group_entry` cannot have any backups at all. Fallback ports for
+multicast uses one of the backups(i) replicas when the primary `port` of a
+replicas(i) becomes unavailable.
```

Comment:
Is the intent that when a primary port goes down, that the corresponding backup ports must be tried in the order specified, and the system must use the first one in the list that is up?

Or do you intend that it is explicitly _unspecified_ which backup replica is used by an implementation, as long as the one selected has a port that is up?

Reply:
The intent is that when the primary port goes down, the system will use the first one in the list that is up. In this case order does indeed matter